docs/data/tree-view/getting-started/getting-started.md

```diff
@@ -13,25 +13,11 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 ## Installation
 
-Using your favorite package manager, install `@mui/x-tree-view`:
+Using your favorite package manager, install `@mui/x-tree-view-pro` for the commercial version, or `@mui/x-tree-view` for the free community version.
 
 <!-- #default-branch-switch -->
 
-<codeblock storageKey="package-manager">
-
-```bash npm
-npm install @mui/x-tree-view
-```
-
-```bash pnpm
```

Comment:
Nitpick: The `styled-engine` section below has `yarn` as the second choice.
It would be nice to align and put it last. 😉